esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -1865,41 +1876,52 @@ bool CFileSprayEx::onSprayFixed(IEspContext &context, IEspSprayFixed &req, IEspS
     try
     {
         context.ensureFeatureAccess(FILE_SPRAY_URL, SecAccess_Write, ECLWATCH_FILE_SPRAY_ACCESS_DENIED, "Failed to do Spray. Permission denied.");
-
         StringBuffer destFolder, destTitle, defaultFolder, defaultReplicateFolder;
+        StringBuffer gName, ipAddr;
+        bool useStoragePlane = false;
 
         const char* destNodeGroup = req.getDestGroup();
-        if(destNodeGroup == NULL || *destNodeGroup == '\0')
-            throw MakeStringException(ECLWATCH_INVALID_INPUT, "Destination node group not specified.");
+        const char* destPlane = req.getDestPlane();
+        if (!isEmptyString(destPlane))
```

Comment:
What functionality does destPlane provide?  I think it is identical to destCluster.

Reply:
It is the same at the moment. Added for consistency.  I can remove in the service but would you want it to be removed from dfuplus too?

Reply:
I don't think it is worth having two parameters to the service which do the same thing - it is adding complexity for no real benefit.  I don't have a strong opinion on the dfuplus command.  I suspect it  is simpler without.  At some point all the parameters could do with reviewing, but probably not now...